File: tests/python/cases/case_221_local_clone_named_retained_02.py
"""
Task case 221: local_clone_named_retained_02 deep stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_221_local_clone_named_retained_02(input_data):
    task_name = 'local_clone_named_retained_02'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'extreme',
        'stage': 'stress',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    retained_local = [dict(payload)]
    _ = retained_local[0]['task']
    # SAFE: misleading local name but no global retention.
    return payload['checksum']

File: tests/nodejs/cases/case_221_local_clone_named_retained_02.js
/**
 * Task case 221: local_clone_named_retained_02 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case221LocalCloneNamedRetained02(input) {
  const taskName = 'local_clone_named_retained_02';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const retainedLocal = [payload];
  void retainedLocal.length;
  // SAFE: local retained* name only.
  return payload.checksum;
}

module.exports = { case221LocalCloneNamedRetained02 };
